src/averageAges.js

```diff
@@ -14,12 +14,29 @@
  *
  * @return {number}
  */
+
+function averageAge(ages) {
```

Comment:
function names start with a verb - it must be clear we're dealing with a function